builder/builder-next/adapters/snapshot/snapshot.go

```diff
@@ -480,6 +480,9 @@ func (m *mountable) Release() error {
 	}
 
 	m.mounts = nil
+	defer func() {
+		m.release = nil
```

Comment:
wondering; should it check the returned error? (i.e., only do this if the `release()` was successful?)

Reply:
looking at different drivers there doesn't seem to be consistency in what we can assume about the mount state after an error appears here.